xchange-coindeal/src/main/java/org/knowm/xchange/coindeal/CoindealAdapters.java

```diff
@@ -0,0 +1,37 @@
+package org.knowm.xchange.coindeal;
+
+import org.knowm.xchange.coindeal.dto.marketdata.CoindealOrderBook;
+import org.knowm.xchange.currency.CurrencyPair;
+import org.knowm.xchange.dto.Order;
+import org.knowm.xchange.dto.marketdata.OrderBook;
+import org.knowm.xchange.dto.trade.LimitOrder;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public final class CoindealAdapters {
+
+    /** private Constructor */
+    private CoindealAdapters() {}
+
+    public static OrderBook adaptOrderBook(CoindealOrderBook coindealOrderBook, CurrencyPair currencyPair){
+        List<LimitOrder> asks = new ArrayList<>();
+            coindealOrderBook.getAsks().forEach(coindealOrderBookEntry -> {
```

Comment:
did you build this using maven?
I should have fixed formatting problems like this

Reply:
Yeah the main pom is still not formatted correctly. Can you run `mvn com.coveo:fmt-maven-plugin:format`?